Automattic/jetpack
# Adds connected user information to the Blaze dashboard initial load

This PR changes the initial user information that the Blaze Dashboard uses to load the application. That information will be used by the internal state of the Dashboard, primarily for tracks and analytics.

## Proposed changes:

* Changes the information we are returning in the initial load of the page, to include the connected user info.

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


This module is not open for users yet, we will need to include the Jetpack Blaze privacy information in the future when we open the gate. 

For now, the Dashboard is using the same tracks used in Wordpress.com Calypso (Advertising page).

## Testing instructions:






On a self-hosted site, enable the Blaze Dashboard with `add_filter( 'jetpack_blaze_dashboard_enable', '__return_true' );`

* Go to Tools->Advertising
* Verify that the Jetpack Blaze dashboard loads correctly

![Screenshot 2023-06-30 at 11 00 35](https://github.com/Automattic/jetpack/assets/1258162/dd4b0a7d-bbd0-4db8-9e2b-a67228c78e54)
